packages/datagateway-download/src/downloadCart/downloadCartApi.ts

```diff
@@ -157,8 +181,17 @@ export const downloadPreparedCart: (
 
 export const getSize: (
   entityId: number,
-  entityType: string
-) => Promise<number> = (entityId: number, entityType: string) => {
+  entityType: string,
+  facilityName: string,
+  apiUrl: string,
+  downloadApiUrl: string
```

Comment:
It might be clearer, especially in cases that require multiple settings values, to supply the values from settings in an object. e.g. instead, getSize could look like this:

```
getSize(
  entityId: number,
  entityType: string,
  settings: { 
    facilityName: string,
    apiUrl: string,
    downloadApiUrl: string
  }
)
```

It more clearly shows the difference between settings and the regular function arguments. e.g. 
`getSize(1, "dataset", {facilityName: "ISIS", apiUrl: "www.example.com", downloadApiUrl: "www.example.com"})`